### DISCUSSION TITLE
How to update

### DISCUSSION THREAD

**Discussion by y0sh1ne** [spacedriveapp/spacedrive#2277] [Community Support] [answered]
It told me I need to update the application this morning, and I just click that upate button.
Then it told me to restart the Spacedrive for updating, but it not worked, it still show that messege for updating. 
I'm 0.2.8 now.

**Comment by utkubakir** [comment]
Can you record a video?

**Reply by y0sh1ne** [reply, marked as answer]
It just worked when I tried to record video. I have no idea about what happened, maybe it's because I restarted the system. Sorry for bothering.

By the way, It showed a message window (the same UI with the one which remind me to update) after downloading. And the messege context is something like "✅you should restart spacedrive to install" (✅ is round, I remember this clearly). But this time, it automatically pops up the installation window before I saw that message.

**Reply by utkubakir** [reply]
Oh wow, yeah might have been a faulty download or something. Glad everything worked out
